app/views/landing_page/landing_page.erb

```diff
@@ -35,6 +35,10 @@
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
 
+  <% if render_analytics %>
```

Comment:
Correct, it's for the editor. I didn't know how to test existence of the local, thanks!